src/core/default-query.ts

```diff
@@ -0,0 +1,25 @@
+import {getTypeRepresentation} from './type-representation';
+import {InteractionCreator} from './interaction-creator';
+import * as tsMorph from 'ts-morph';
+import {changeObjectRepresentationIntoExample} from './create-pact-example-object';
+import {changeObjectRepresentationIntoMatchingRules, MatchingRules} from './create-pact-matching-rules';
+import {PACT_ANNOTATIONS} from '../consts/pact-annotations';
+import qs, {IStringifyOptions} from 'qs';
+
+export class DefaultQuery {
```

Comment:
I would remove "Default" prefixes for these classes. Sounds like some default value for query/body.